.vscode/settings.json

```diff
@@ -8,5 +8,9 @@
     },
     "typescript.tsdk": "./node_modules/typescript/lib", // we want to use the TS server from our node_modules folder to control its version
     "typescript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": true,
-    "typescript.format.insertSpaceAfterOpeningAndBeforeClosingEmptyBraces": false
+    "typescript.format.insertSpaceAfterOpeningAndBeforeClosingEmptyBraces": false,
+    "cmake.sourceDirectory": "${workspaceFolder}/test/fakeOutputGenerator",
```

Comment:
What is this for? Is the fakeOutputGenerator test like a decoy so the extension doesn't prompt or do any significant work in CI?

Reply:
It pretends to be a compiler and spits out fake version information when we launch it.